<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBWindow</class>
 <widget class="QMainWindow" name="DBWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Db Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDBName">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>txtDBName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtServerName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>txtServerName</string>
    </property>
    <property name="text">
     <string>(localdb)\\MSSQLLocalDB</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStringConnection">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>txtStringConnection</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>StringConection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnectionStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string>lblConnectionStatus</string>
    </property>
    <property name="text">
     <string>Connection:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>811</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
